cypress/integration/logistics/materialentnahme_complete_HUs.js

```diff
@@ -0,0 +1,178 @@
+import { BPartner } from '../../support/utils/bpartner';
+import { DiscountSchema } from '../../support/utils/discountschema';
+import { ProductCategory } from '../../support/utils/product';
+import { Builder } from '../../support/utils/builder';
+import { appendHumanReadableNow } from '../../support/utils/utils';
+import { Warehouse } from '../../support/utils/warehouse';
+import { PurchaseOrder, PurchaseOrderLine } from '../../support/utils/purchase_order';
+import {
+  applyFilters,
+  selectNotFrequentFilterWidget,
+  toggleNotFrequentFilters,
+  clearNotFrequentFilters,
+} from '../../support/functions';
+
+let productForPackingMaterial;
+let packingInstructionsName;
+let productName1;
+let productCategoryName;
+let discountSchemaName;
+let priceSystemName;
+let priceListName;
+let priceListVersionName;
+let productType;
+let vendorName;
+let warehouseName;
+let quantity;
+let quantityTu;
+let materialentnahme;
+let husAfterMoving;
+
+it('Read the fixture', function() {
+  cy.fixture('logistics/materialentnahme_complete_HUs.json').then(f => {
+    productForPackingMaterial = appendHumanReadableNow(f['productForPackingMaterial']);
+    packingInstructionsName = appendHumanReadableNow(f['packingInstructionsName']);
+    productName1 = appendHumanReadableNow(f['productName1']);
+    productCategoryName = appendHumanReadableNow(f['productCategoryName']);
+    discountSchemaName = appendHumanReadableNow(f['discountSchemaName']);
+    priceSystemName = appendHumanReadableNow(f['priceSystemName']);
+    priceListName = appendHumanReadableNow(f['priceListName']);
+    priceListVersionName = appendHumanReadableNow(f['priceListVersionName']);
+    productType = f['productType'];
+    vendorName = appendHumanReadableNow(f['vendorName']);
+    materialentnahme = f['materialentnahme'];
+    warehouseName = appendHumanReadableNow(f['warehouseName']);
+    quantity = f['quantity'];
+    quantityTu = f['quantityTu'];
+    husAfterMoving = f['husAfterMoving'];
+  });
+});
+
+it('Create  warehouse', function() {
```

Comment:
Any particular reason why we have all the test prerequisites creation in separate test cases instead of just one ?

Reply:
Some times, in the pre-requisites there are some errors occurring and if they are split into multiple tests, it's easier to spot exactly at which step the test has failed. For clarity too. We changed this for a while now. Also, for the tests which need 2 or 3 products to be created, if you put all of them in the same it...the test fails.